javascript/javascript3/week1/Giphy/giphy.js

```diff
@@ -0,0 +1,31 @@
+const apiKey = "6om4RdQYOfus5aHsF55R28uIQDtB3CRt";
+
+function renderData(data) {
+  const root = document.getElementById("root");
+  const list = document.createElement("ul");
+  root.childNodes.forEach((child) => {
+    root.removeChild(child);
+  });
+  root.appendChild(list);
+  data.forEach((img) => {
+    const renderImg = document.createElement("img");
+    renderImg.src = img.images.preview_gif.url;
+    const item = document.createElement("li");
+    item.appendChild(renderImg);
+    list.appendChild(item);
+  });
+}
+
+function fetchData() {
+  const inputGif = document.getElementById("gifReq");
+  const getGif = inputGif.value;
+  let totalGif = document.getElementById("totalGif");
+  let maxView = totalGif.value;
+  fetch(
+    `http://api.giphy.com/v1/gifs/search?q=${getGif}&api_key=${apiKey}&limit=${maxView}`
+  )
+    .then((response) => response.json())
+    .then((response) => {
+      renderData(response.data);
+    });
+}
```

Comment:
Looks all good! Nice and clean code. If you want, you can consider using some comments in your code, for example why you are removing all childNodes (line 6). 
You could also implement some action (like submitting form) when user presses enter after inserting some word into the input field, but it's not a necessity. ;)